llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-subvector.ll

```diff
@@ -238,7 +238,7 @@ define void @extract_v8i1_v64i1_8(ptr %x, ptr %y) {
 ; LMULMAX2-NEXT:    li a2, 32
 ; LMULMAX2-NEXT:    vsetvli zero, a2, e8, m2, ta, ma
 ; LMULMAX2-NEXT:    vlm.v v8, (a0)
-; LMULMAX2-NEXT:    vsetivli zero, 1, e8, mf4, ta, ma
+; LMULMAX2-NEXT:    vsetivli zero, 1, e8, mf8, ta, ma
```

Comment:
Ah, you don't want this.  This is not a profitable change.

We don't want "minimum LMUL".  We want "minimum LMUL which is a multiple of DLEN".